signature/jws/jwt_test.go

```diff
@@ -0,0 +1,126 @@
+package jws
+
+import (
+	"crypto"
+	"crypto/x509"
+	"errors"
+	"testing"
+
+	"github.com/notaryproject/notation-core-go/signature"
+	"github.com/notaryproject/notation-core-go/testhelper"
+)
+
+type errorLocalSigner struct {
+	algType      signature.KeyType
+	size         int
+	keySpecError error
+}
+
+// Sign returns error
+func (s *errorLocalSigner) Sign(payload []byte) ([]byte, []*x509.Certificate, error) {
+	return nil, nil, errors.New("sign error")
+}
+
+// KeySpec returns the key specification.
+func (s *errorLocalSigner) KeySpec() (signature.KeySpec, error) {
+	return signature.KeySpec{
+		Type: s.algType,
+		Size: s.size,
+	}, s.keySpecError
+}
+
+// PrivateKey returns nil.
+func (s *errorLocalSigner) PrivateKey() crypto.PrivateKey {
+	return nil
+}
+
+// CertificateChain returns nil.
+func (s *errorLocalSigner) CertificateChain() ([]*x509.Certificate, error) {
+	return nil, nil
+}
+
+func Test_remoteSigningMethod_Verify(t *testing.T) {
+	defer func() {
+		if d := recover(); d == nil {
+			t.Fatal("should panic")
+		}
+	}()
+	s := &remoteSigningMethod{} // Sign signs the payload and returns the raw signature and certificates.
+	s.Verify("", "", nil)
```

Comment:
This recover will always return a non-nil error
```suggestion
	defer func() {
		if d := recover(); d == nil {
			t.Fatal("should have error")
		}
	}()
	s.Verify("", "", nil)
	t.Errorf("should have panic")
```

Reply:
Changed the Verify() function to return an error.